.eslintrc.js

```diff
@@ -18,8 +18,24 @@ var path = require('path');
 
 module.exports = {
   root: true,
-  plugins: ['@spotify', 'notice', 'react', 'testing-library'],
+  plugins: ['@spotify', 'notice', 'react', 'testing-library', '@backstage'],
   rules: {
+    '@backstage/no-mixed-plugin-imports': [
+      'error',
+      {
+        // TODO: Fix these either by right role or by moving things to new packages
+        excludedTargetPackages: [
+          '@backstage/test-utils',
```

Comment:
These ones have actual issues in the source code (which are also in exception in the `verify-local-dependencies.js`). Some of them could be fixed, some of them maybe not